game/environment.go

```diff
@@ -76,7 +88,9 @@ type VPNConfig struct {
 	Endpoint         string
 }
 
-func NewEnvironment(conf *GameConfig, vlib vbox.Library) (*GameConfig, error) {
+// todo: this needs to be fixed and corrected
```

Comment:
Explanation on this TODO?

Reply:
Hmm, the function is  taking GameConfig as parameter and returns it by adding some stuff, which is very rare usage of functions in general, i would expect to get the GameConfig as parameter and return a Game instead of Gameconfig again.

Reply:
Makes sense, feel free to modify it otherwise create an issue :)